packages/mds-geography-author/api.ts

```diff
@@ -261,46 +261,47 @@ function api(app: express.Express): express.Express {
             if (writeErr instanceof DependencyMissingError) {
               return res.status(404).send({ error: writeErr })
             }
-            return next(new ServerError())
+            return next(writeErr)
           }
         } else {
-          return next(new ServerError(updateErr))
+          return next(updateErr)
         }
       }
     }
   )
 
-  app.put(
-    pathPrefix('/geographies/:geography_id/publish'),
-    checkGeographyAuthorApiAccess(scopes => scopes.includes('geographies:publish')),
-    async (
-      req: GeographyAuthorApiPublishGeographyRequest,
-      res: GeographyAuthorApiPublishGeographyResponse,
-      next: express.NextFunction
-    ) => {
-      const { geography_id } = req.params
-      try {
-        await db.publishGeography({ geography_id })
-        const published_geo = await db.readSingleGeography(geography_id)
-        return res.status(200).send({ version: res.locals.version, data: { geography: published_geo } })
-      } catch (updateErr) {
-        if (updateErr instanceof NotFoundError) {
-          return res.status(404).send({ error: `unable to find geography of ${geography_id}` })
+  app
+    .put(
+      pathPrefix('/geographies/:geography_id/publish'),
+      checkGeographyAuthorApiAccess(scopes => scopes.includes('geographies:publish')),
+      async (
+        req: GeographyAuthorApiPublishGeographyRequest,
+        res: GeographyAuthorApiPublishGeographyResponse,
+        next: express.NextFunction
+      ) => {
+        const { geography_id } = req.params
+        try {
+          await db.publishGeography({ geography_id })
+          const published_geo = await db.readSingleGeography(geography_id)
+          return res.status(200).send({ version: res.locals.version, data: { geography: published_geo } })
+        } catch (updateErr) {
+          if (updateErr instanceof NotFoundError) {
+            return res.status(404).send({ error: `unable to find geography of ${geography_id}` })
+          }
+          return next(new ServerError(updateErr))
         }
-        return next(new ServerError(updateErr))
       }
-    }
-  )
-
-  app.use(async (error: Error, req: ApiRequest, res: ApiResponse, next: NextFunction) => {
-    const { method, originalUrl } = req
-    GeographyAuthorLogger.error('Fatal MDS Geography Author Error (global error handling middleware)', {
-      method,
-      originalUrl,
-      error
-    })
-    return res.status(500).send({ error })
-  })
+    )
+    .use(ApiErrorHandlingMiddleware)
+  // app.use(async (error: Error, req: ApiRequest, res: ApiResponse, next: NextFunction) => {
```

Comment:
Why not delete this code if it's superfluous?

Reply:
thanks!!